domain/src/main/java/gov/cms/ab2d/domain/User.java

```diff
@@ -27,4 +40,23 @@
     @ManyToOne
     @JoinColumn(name = "sponsor_id")
     private Sponsor sponsor;
+
+    @ManyToMany(cascade = { PERSIST, MERGE })
+    @JoinTable(
+            name = "user_role",
+            joinColumns = @JoinColumn(name = "user_account_id"),
+            inverseJoinColumns = @JoinColumn(name = "role_id"))
+    private Set<Role> roles = new HashSet<>();
+
+
+    public void addRole(Role role) {
```

Comment:
Next two methods will change based on the previous comment

Reply:
done